scst-store/amr/graphql-query.hbs.md

```diff
@@ -3,15 +3,83 @@
 This topic tells you how to connect to the GraphQL playground and how to query each supported data model.
 For information about the AMR data models, see [AMR Data Models](data-model-and-concepts.hbs.md).
 
-## <a id='connecting-to-graphql'></a> Connecting to GraphQL playground
+## <a id='connecting-to-graphql'></a> Connecting to AMR GraphQL
+
+There are two ways you can perform GraphQL queries:
+
+- Using the GraphQL playground
+- Using [cURL](https://curl.se/)
 
 Vmware recommends enabling ingress. The Supply Chain Security
 Tools for Tanzu – Store and Artifact Metadata Repository (AMR) packages share
 the same ingress configuration. Setting `ingress_domain` for the SCST -
```

Comment:
🔧 
Can you please change `Setting ingress_domain for the SCST - Store applies the same value for AMR` to `Same `ingress_domain` is shared between SCST - Store and AMR`

Reply:
The phrases are a bit different. The current phrase explains that they don't need to set the ingress value in the TAP values file for AMR, as it will use the one from the SCST - Store when they set it in the TAP values file.

The updated string explains the values are shared but does not explain that setting the value in the SCST - Store section of the TAP values will also apply it to the AMR.